### PR TITLE
[Grav] Improve README and writing documentation page

### PR DESCRIPTION
**Pull request by tituspijean**
- [x] README
- [x] Writing documentation guidelines (like, only put the alias/route without the language code, use Markown, available plugins, how to set up your own server, ...)
- [x] Add information on specific pages ( like helpers, cf. https://github.com/YunoHost/doc/pull/1563#issuecomment-779131818)
- [x] How to set up the Grav and Git sync plugins
    - [x] In `config/themes/yunohost-docs.yaml`, set the `commits` and `tree` keys

### CODE REVIEW

File: pages/04.contribute/02.write_documentation/write_documentation.md [hunk 1]
```diff
@@ -9,22 +9,142 @@ routes:
 
 ! This page is outdated and should be reworked
```
Comment (alexAubin) [inline pages/04.contribute/02.write_documentation/write_documentation.md:10]:
```suggestion
```

**Review by alexAubin — COMMENTED** [review]
Shouldn't we delete the fr counterpart until somebody re-translates the page ?

**Comment by tituspijean** [comment]
@alexAubin, done!

**Comment by alexAubin** [comment]
Ah tu as traduit du coup :D Ben cool ! Yolomerging !

**Pull request merged**